src/services/configService.test.ts

```diff
@@ -113,6 +114,12 @@ describe("Config Service", () => {
       const service = new ConfigService(sls, {} as any);
       expect(service.getRegion()).toEqual(location);
     });
+
+    it("use os property from SLS yml config", () => {
```

Comment:
can we combine this test with the one in line 134? to me they're testing the same functionality

Reply:
The test on 134 is testing whether or not the `external` flag is set, which is different than a default linux deployment. The default is to remote build. External means you want to take care of the trigger syncing yourself (remote build is the preferred option)